LUFA/Drivers/USB/Class/Common/HIDClassCommon.h

```diff
@@ -330,6 +330,23 @@
 		#define HID_KEYBOARD_SC_RIGHT_SHIFT                       0xE5
 		#define HID_KEYBOARD_SC_RIGHT_ALT                         0xE6
 		#define HID_KEYBOARD_SC_RIGHT_GUI                         0xE7
+		#define HID_KEYBOARD_SC_MEDIA_PLAY                        0xE8
```

Comment:
Sounds like a plan. We can never hope to cover all the usage pages/usages, but I think there's a well-defined sub-set of them that most people actually want (i.e. the ones on a typical full function keyboard with media extensions). A new namespace would be a good idea -- migration won't be difficult for end-users.